tests/trestle/core/markdown/markdown_node_test.py

```diff
@@ -187,3 +187,14 @@ def test_modify_subtree(testdata_dir: pathlib.Path, tmp_trestle_dir: pathlib.Pat
     assert len(list(subtree.get_all_headers_for_level(3))) == 4
     assert len(list(subtree.get_all_headers_for_level(4))) == 4
     assert subtree.get_node_header_lvl() == 1
+
+
+def test_get_header_level() -> None:
+    """Test get header level."""
+    node = MarkdownNode('foo', 'bar', 0)
```

Comment:
SectionContent() instead of 'bar' to make typing happy

Reply:
done